tasks/src/robocorp/tasks/cli.py

```diff
@@ -9,18 +9,36 @@
 code which must be automatically logged is not imported prior the the `cli.main`
 call.
 """
-
+import os
 import sys
+import warnings
 from typing import List, Optional, Protocol
 
-# Use certificates from native storage (if `truststore` installed)
-if sys.version_info >= (3, 10):
-    try:
-        import truststore  # type: ignore
 
-        truststore.inject_into_ssl()
-    except ModuleNotFoundError:
-        pass
+def inject_truststore():
+    # Use certificates from native storage (if `truststore` installed)
+
+    # RC_USE_TRUSTSTORE env var is only used to avoid unwanted warnings in tests
+    # and should not be used for other purposes
+    if os.getenv("RC_USE_TRUSTSTORE", "True").lower() in ["false", "0"]:
+        return
```

Comment:
```suggestion
    # `_RC_TEST_USE_TRUSTSTORE` env var is only used to avoid unwanted warnings in tests
    #  and should not be used for other purposes outside of its internal usage.
    if os.getenv("_RC_TEST_USE_TRUSTSTORE", "True").lower() in ["false", "0"]:
        return
```
Added a leading underscore (`_`) to mark it as "private" and placed inside a `TEST` word to somehow mark it as required during testing.

Feel free to pick-up anything from the ideas above in case you don't want external users to start fiddle with `RC_USE_TRUSTSTORE`, but if we expect to control the behaviour publicly through this env var, then the name is already perfect.

Reply:
I like that!